src/__tests__/operate.test.js

```diff
@@ -0,0 +1,77 @@
+import operate from '../logic/operate';
+
+describe('Supported calculator operators', () => {
+  test('2 + 2 = 4', () => {
+    const a = 2;
+    const b = 2;
```

Comment:
- [ ] You also have the operate.test.js duplicated in multiple directories; `tests` and `__tests__`, please remove one of the test files.